.gitlab-ci.yml

```diff
@@ -71,54 +71,115 @@ release:
         - "*/target/surefire-reports/TEST-*.xml"
         - "*/target/failsafe-reports/TEST-*.xml"
 
-whois-groovy-test:
+
+.itest_template: &itest_configuration
   <<: *it_env
```

Comment:
do we really need a new template? Can it be merged with existing one it_template?